checkbox-support/checkbox_support/dbus/gnome_monitor.py

```diff
@@ -104,6 +106,107 @@ def set_extended_mode(self) -> Dict[str, str]:
         self._apply_monitors_config(state[0], extended_logical_monitors)
         return configuration
 
+    def cycler(
+        self,
+        res: bool = True,
+        max_res_amount: int = 5,
+        trans: bool = False,
+        log: Callable[..., Any] = None,
+        action: Callable[..., Any] = None,
+        **kwargs
+    ):
+        """
+        Cycling change the monitors configurations automatically
+
+        Args:
+            res: Cycling the resolution or not.
```

Comment:
This function is used to replace `id: graphics/{index}_resolution-change_{product_slug}` and `id: graphics/{index}_rotation_{product_slug}`. Therefore, set the `res` to `false` could cycle rotations only. However, you are right, it could be controlled by `max_res_amount`. How about changing the resolution_filter to a callable function ?

Reply:
Oh ok, considering we can still cycle through transformation it makes sense to have the switch. I'd just recommend using the `resolution` name, to make it clearer.